drivers/clock_control/Kconfig.npcx

```diff
@@ -68,3 +68,23 @@ config CLOCK_NPCX_APB3_PRESCALER
 	  12.5MHz <= APB3_CLK <= 50MHz.
 	  - The frequency of APB3_CLK must be an integer division (including 1)
 	  of the frequency of the Core clock.
+	  APB3 prescaler, allowed values: From 1 to 10.
+
+config CLOCK_NPCX_APB4_PRESCALER
```

Comment:
Hey not quite a comment for this specific change but I've seen some work happening on another platform to move the prescaler settings off the config and into the device tree (https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/clock/st%2Cstm32f1-pll-clock.yaml). The result looks nice, something worth considering if you end up doing more work on the clock code.

Reply:
It seems nice. Nuvoton will add another PR for this feature (moving PLL prescaler settings from Kconfig to the dts file). Is it OK with you?

Reply:
Oh yeah definitely.

Reply:
Hi @fabiobaltieri:

Nuvoton has submitted a PR, #36217, for this request. Please take a look and any comment is appreciated. Thanks.